docs/ndcubesequence.rst

```diff
@@ -129,12 +129,16 @@ more details on these features.
 Dimensions
 ----------
 
-Analagous to `ndcube.NDCube.dimensions`, there is also a
-`~ndcube.NDCubeSequence.dimensions` property for easily inspecting the
-size and shape of an `~ndcube.NDCubeSequence` instance::
+Analagous to `ndcube.NDCube.dimensions`, there are also
+`~ndcube.NDCubeSequence.dimensions` and
+`~ndcube.NDCubeSequence.world_axis_physical_types` properties for
+easily inspecting the shape and phsyical axis types of an
+`~ndcube.NDCubeSequence` instance::
 
   >>> my_sequence.dimensions
-  SequenceDimensionPair(shape=(3, <Quantity 3. pix>, <Quantity 4. pix>, <Quantity 5. pix>), axis_types=('Sequence Axis', 'HPLN-TAN', 'HPLT-TAN', 'WAVE'))
+  (<Quantity 3. pix>, <Quantity 3. pix>, <Quantity 4. pix>, <Quantity 5. pix>)
+  >>> my_sequence.world_axis_physical_types
+  ('custom:Sequence Axis', 'custom:pos.helioprojective.lon', 'custom:pos.helioprojective.lat', 'em.wl')
```

Comment:
I don't like `custom:Sequence Axis` we should make it closer to the general form of the UCD1+ vocab, i.e. `thing.thing`.

Reply:
`pos.obs.sequence`

Reply:
Agreed.  Changed it to ```meta.obs.sequence```.